webrtc.html

```diff
@@ -517,6 +517,10 @@
 
         <ol>
           <li>
+            <p>Let <var>p</var> be a new promise</p>.
+          </li>
+
+          <li>
             <p>Append an object representing the current call being handled
```

Comment:
As for "execute the function", that's what was already there.  All I did was remove the word "asynchronously".  Nevertheless, I will change it to "object" to make it more consistent.